tensorflow/core/kernels/stateful_random_ops_cpu_gpu.h

```diff
@@ -86,12 +86,19 @@ using CPUDevice = Eigen::ThreadPoolDevice;
 
 using GPUDevice = Eigen::GpuDevice;
 
+struct UpdateVariableAndFill_Philox_Arg {
+  int64 output_size;
+  int64 alg_tag_skip;
+  ScopedUnlockUnrefVar* not_used;
```

Comment:
Why is this not called state_var_guard? It is used after all (`state_var_guard->Release();`).

Reply:
It is not used by the GPU kernel.